### PR TITLE
Updates Brain Trauma Messages

### PR DESCRIPTION
**Pull request by meemofcourse**
## About The Pull Request

Title. Makes it more shiptest related.

## Why It's Good For The Game

heho funny brain damage

## Changelog

:cl:
tweak: Updated brain trauma messages
/:cl:

### CODE REVIEW

File: strings/traumas.json [hunk 1]
```diff
@@ -13,59 +12,75 @@
...
 		"PACKETS!!!",
 		"port ba@pick(y_replacements) med!!!!",
-		"REVIRT COBY CHEM!!!!!!!!",
 		"youed call her a toeugh bithc",
 		"closd for merbegging",
 		"@pick(semicolon)pray can u @pick(create_verbs) @pick(create_nouns)???",
 		"GEY AWAY FROM ME U GREIFING PRICK!!!!",
-		"ur a fuckeing autist!",
-		"@pick(semicolon)HELP SHITECIRTY MURDERIN  MEE!!!",
+		"@pick(semicolon)HELP SHITECIRTY MURDERIN MEE!!!",
```
Comment (Erikafox) [inline strings/traumas.json]:
```suggestion
		"@pick(semicolon)HELP INTEG MURDERIN MEE!!!",
```

File: strings/traumas.json [hunk 2]
```diff
@@ -13,59 +12,75 @@
...
+		"*monch",
+		"Amonger",
+		"Live mas",
+		"top ten goliath funny moments",
+		"put me... in a stew... then buryy mee...",
+		"@pick(semicolon)GTT AWYA FROM HER YOU BITHC!!",
+		"@pick(semicolon)IS TATH A FUCKIGN,, MOTH.",
+		"@pick(semicolon)BRAZIL NUMEOR UNO...!!!!",
+		"@pick(semicolon)blbue hair??? I'M GOIgN TO KILL THDT FUIKNG COW!",
+		"@pick(semicolon)HOW DO I @pick(ghetto)?????"
```
Comment (Erikafox) [inline strings/traumas.json]:
```suggestion
		"@pick(semicolon)HOW DO I @pick(ghetto)?????",
		"ough",
		"ourgh",
		"He's BALD",
		"I am normal and can be trusted with industrial chemicals",
		"Welcome to the bathroom",
		"THE AMOUNGS BROKE ALL MY BONES"
		
```
Not a single Nyaru line? For shame.

File: strings/traumas.json [hunk 3]
```diff
@@ -79,24 +94,12 @@
 		"stun gloves"
 	],
 
-	"george": ["joerge", "george", "gorge", "gdoruge"],
+	"john": ["joehn", "jonn", "jouhn", "jeeoun"],
```
Comment (Erikafox) [inline strings/traumas.json:124]: joehrn

File: strings/traumas.json (hunk 2 above)
Comment (meemofcourse) [reply strings/traumas.json]: i wasn't around when Nyaru was popular

Comment (Erikafox) [reply strings/traumas.json]: neither was I but it's like heritage at this point

File: strings/traumas.json [hunk 4]
```diff
@@ -13,59 +12,102 @@
...
+		"butT ShIPPtSt nEVER was a WrAZnOe liKE thaEt!!",
+		"A deEoP-setED dISEuire... fuR pERfECTiOsM...",
+		"HUMORER IS A DECLIAT THINGE!!",
+		"who's the asshole flying the pill class",
+		"IVOR@pick(y_replacements) WHAT THE FOUCKE ARE U DUING!!!??",
+		"RESIEST BIG MOETH",
+		"WE MUSTE RIASE UOP AGANST BEEG MOTNH",
+		"Luckily, I passed high school physics",
+		"I WANT NOTHING MORE IN THIS LIFE THAN TO CUDDLE UP WITH A CUTE MOTH WAIFU!!",
+		"GO TO HORNY JAIL!!"
```
Comment (axelzonvolt) [inline strings/traumas.json:110]: Can we get at least one line about cement consumption in here?

File: strings/traumas.json [hunk 5]
```diff
@@ -89,14 +89,25 @@
...
-		"WHER.E SHIPQUEST???,???"
+		"WHER.E SHIPQUEST???,???",
+		"ei,,ither wAy... It Is WHt ti is",
+		"butT ShIPPtSt nEVER was a WrAZnOe liKE thaEt!!",
+		"A deEoP-setED dISEuire... fuR pERfECTiOsM...",
+		"HUMORER IS A DECLIAT THINGE!!",
+		"who's the asshole flying the pill class",
+		"IVOR@pick(y_replacements) WHAT THE FOUCKE ARE U DUING!!!??",
+		"RESIEST BIG MOETH",
+		"WE MUSTE RIASE UOP AGANST BEEG MOTNH",
```
Comment (Anderson-Cooper) [inline strings/traumas.json:107]: this reminds me from when i was a shitter in the shiptest discord

the glory days

**Review by MarkSuckerberg — APPROVED** [review]
brings a tear to my eye

**Pull request merged**